CNV/cnv-2.1.0.sh

```diff
@@ -27,28 +28,30 @@ if [ "${CLUSTER}" == "KUBERNETES" ]; then
     MARKETPLACE_NAMESPACE="marketplace"
 fi
 
-if [ -z "${QUAY_USERNAME}" ]; then
-    echo "QUAY_USERNAME"
-    read QUAY_USERNAME
-fi
+if [ -z "${TOKEN}" ]; then
+    if [ -z "${QUAY_USERNAME}" ]; then
+	echo "QUAY_USERNAME"
+	read QUAY_USERNAME
```

Comment:
Being interactive is probably not a good idea, this could be an hour after the install was started.

It was previously suggest to exit out if we don't have valid values .